src/reactive/keystone_saml_mellon_handlers.py

```diff
@@ -13,51 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import uuid
-
 # import to trigger openstack charm metaclass init
-import charm.openstack.keystone_saml_mellon # noqa
+import charm.openstack.keystone_saml_mellon as keystone_saml_mellon  # noqa
```

Comment:
You could do this instead:

    import charms_openstack.bus
    charms_openstack.bus.discover()

Reply:
Yes, this is a cool feature that @gnuoy implemented.  It does auto-discovery of all charms.openstack related functionality in a charm, regardless of where it is located.

Reply:
Done.